### PR TITLE
rewording / add howto use labels/tags

### PR DESCRIPTION
**Pull request by jessegoodier**
adding some pictures and rewording. 

Happy to use the previous wording if you feel it is better.

### CODE REVIEW

File: using-kubecost/navigating-the-kubecost-ui/saved-reports/cost-center-report.md [hunk 1]
```diff
@@ -32,6 +36,14 @@ Your cost center should automatically appear in the Report Preview. There is no
 
 After configuring a cost center, you can select _Collapse_ to close that configuration (this is only to condense the page view, it will not affect your overall reported data).
 
+### Tags and labels
+
+Any cloud provider tag or label can be used, be sure to follow the [respective guide](https://docs.kubecost.com/install-and-configure/install/cloud-integration) to ensure that they are included with the billing data.
```
Comment (thomasvn) [inline using-kubecost/navigating-the-kubecost-ui/saved-reports/cost-center-report.md:41]:
```suggestion
Any cloud provider tag or label can be used, be sure to follow the [respective guide](./cloud-integration.md) to ensure that they are included with the billing data.
```

Local hyperlink instead?

**Comment by thomasvn** [comment]
Looks good, screenshots really helpful!

**Review by thomasvn — DISMISSED** [review]

**Comment by thomasvn** [comment]
@bstuder99 For final review?

**Comment by jessegoodier** [comment]
> @bstuder99 For final review?

for sure. Feel free to merge if you are good with the edits @bstuder99

File: using-kubecost/navigating-the-kubecost-ui/saved-reports/cost-center-report.md [hunk 2]
```diff
@@ -1,10 +1,14 @@
 # Cost Center Report
 
 {% hint style="warning" %}
-Cost Center Report is a beta feature. Read the documentation carefully.
+Cost Center Report is a beta feature. Please share your feedback as we are in active development of this feature.
```
Comment (bstuder99) [inline using-kubecost/navigating-the-kubecost-ui/saved-reports/cost-center-report.md:4]: The original text is how we describe all beta features and I'd like to leave that as is. "Beta" implies the feature is in active development and is receptive to user feedback, correct? Want to get @kwombach12's feedback here

Comment (jessegoodier) [reply using-kubecost/navigating-the-kubecost-ui/saved-reports/cost-center-report.md:4]: I didn't see your comment until just now. Let me know and I can re submit.

Comment (bstuder99) [reply using-kubecost/navigating-the-kubecost-ui/saved-reports/cost-center-report.md:4]: No worries, I was out for a few days and didn't see this. The rest of the changes look good so we can roll this out while we wait for a response.

**Review by bstuder99 — APPROVED** [review]

**Pull request merged**